extensions/stackdriver/config/v1alpha1/stackdriver_plugin_config.proto

```diff
@@ -45,6 +45,16 @@ message PluginConfig {
   // This is deprecated in favor of AccessLogging enum.
   bool disable_server_access_logging = 1 [deprecated = true];
 
+  // Optional. Allows configuration of the number of log entries request to
+  // batch into a single request. The size is in bytes, so that it allows for
+  // better performance. Default is 4MB. The size of one log entry is approx
+  // 1Kb.
+  int32 max_log_batch_size_in_bytes = 12;
+
+  // Optional. Allows configuration of the time between calls out to the
+  // stackdriver logging service to report new log entries.
+  google.protobuf.Duration log_report_duration = 13;
```

Comment:
These two options sound contradictory. Do we report when the batch size reaches certain limit or when the ticker ticks? Are there many batches in a single request?

Reply:
We flush when the batch size reaches certain limit(configured at 4MB) and export(make grpc call to logging service) the flushed entried when ticker ticks. Yeah there are many log entries batched in a single request.

Reply:
Sorry, I lost track about the context here a bit. Why do we make report duration configurable? Can you add the reason to the description so that we don't forget? Also would the option still apply if we switch to dedicated thread for log reporting?

Also probably better to clarify the comment a bit. Instead of `to report new log entries`, we can make it `to report buffered LogWrite request`.

Reply:
Added.. Specifically to fix https://github.com/istio/istio/issues/21252